### ISSUE TITLE
Missing context in asset tab

### ISSUE THREAD

**Issue by fashxp** [pimcore/studio-ui-bundle#428]
![image](https://github.com/user-attachments/assets/fd3f5c79-f60e-4050-a673-df99c6cea49f)

- open two tabs in parallel
- close one tab --> second tab looses context

**Comment by markus-moser** [comment]
Duplicate of #975 (this one is the original, but the new one has the more detailed description)
